src/backend/marsha/core/api/pairing_challenge.py

```diff
@@ -67,5 +67,8 @@ def pairing_challenge(request):
     Device.objects.get_or_create(id=request.data.get("box_id"))
 
     return Response(
-        {"jitsi_url": f"https://{settings.JITSI_DOMAIN}/{live_pairing.video.id}"}
+        {
+            "jitsi_url": f"https://{settings.JITSI_DOMAIN}/{live_pairing.video.id}",
+            "rtmp_urls": live_pairing.video.get_medialive_input().get("endpoints"),
```

Comment:
In fact there is a problem. This info can be missing, you must check if `video.live_info` is not none.